src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,37 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">
+          {name}
```

Comment:
You repeat this name twice, fix it
![image](https://user-images.githubusercontent.com/91411692/231985796-beba8b49-b01f-4c0d-afac-dce5052ac09d.png)